scripts/google_cloud_mango_install.sh

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env bash
```

Comment:
let's put these in bin/GCE/<all the scripts>

Reply:
done